README.md

```diff
@@ -1,66 +1 @@
-
-#### Minimum Requirements
-* The CLI should be architected using best modularization practices
-* The CLI should require at least three arguments `input-file-path output-file-path transfrom-name` 
-* The CLI should support a minimum of three transforms
-* The CLI should log useful Error messages if used incorrectly
-* The CLI should log a success message on completion
-
-## Testing 
-* Use TDD `describe` and `test` methods to define descriptive tests and increase readablity
-* Each `test` callback should aim to test a small well defined feature of a function
-* Write tests to ensure each function behaves correctly with valid and invalid inputs
-* The CLI should be tested without using `child_process` or any equivilant third party libraries
-
-##  Documentation
-In your README.md describe the exported values of each module you have defined. Every function description should include it's arity (expected number of paramiters), the expected data for each parameter (data-type and limitations), and it's behavior (for both valid and invalid use). Feel free to write any additional information in your README.md.
-
-## Tips
-You will want to define a strategy for solving the problem before you begin to code. Once you have a strategy defined, you can break it down into steps that can be split into helper modules. Each helper module should solve a small specific problem. The main module should utilize the helper modules to execute your original strategy.
-
-###### Example Strategy 
-0. Gather user input (infile, outfile, and transform)
-0. Read the input bitmap file using the fs module 
-0. Parse the bitmap's buffer into object representing a bitmap (using a constructor)
-0. Using metadata from the parsed bitmap object to run a transform on the buffer directly (mutate the colors)
-0. Write the mutated buffer to the output file path
-
-###### Transfrom Ideas
-* Color Palette Transforms 
-  * Invert 
-  * Randomize
-  * Greyscale
-  * Black and White
-  * Darken or Lighten
-  * Add or Mutiply a Hue
-  * Add or Subtract Contrast
-
+The program alters an image (bmp format) using one of three provided transformation types and saves the new image as a different file. Three parameters are required: the file path of the image to be altered (must be a .bmp format), the desired output file path (with .bmp extension), and the name of the transform desired (invert, randomize, or greyscale). If a parameter is not supplied the program will return an error. If the transformation type is not one of the three provided an error will be thrown and no new image will be created.
```

Comment:
all of your files ending in .md should use proper markdown syntax. this is a big project, be a bit more specific :)